docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,187 @@
+| <kbd>Ctrl</kbd> + <kbd>C</kbd><br>or<br><kbd>Ctrl</kbd> + <kbd>Insert</kbd>  | <kbd>Cmd</kbd> + <kbd>C</kbd>       | Copy the cell's contents                                                                              |
+| <kbd>Ctrl</kbd> + <kbd>X</kbd>                                               | <kbd>Cmd</kbd> + <kbd>X</kbd>       | Cut the cell's contents                                                                               |
+| <kbd>Ctrl</kbd> + <kbd>V</kbd><br>or<br><kbd>Shift</kbd> + <kbd>Insert</kbd> | <kbd>Cmd</kbd> + <kbd>V</kbd>       | Paste into the current cell                                                                           |
+| <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                                           | <kbd>Cmd</kbd> + <kbd>Enter</kbd>   | When editing:<br>fill all currently-selected cells with the contents of the currently-edited cell     |
+| <kbd>Ctrl</kbd> + <kbd>Z</kbd>                                               | <kbd>Cmd</kbd> + <kbd>Z</kbd>       | Undo                                                                                                  |
+| <kbd>Ctrl</kbd> + <kbd>Y</kbd>                                               | <kbd>Cmd</kbd> + <kbd>Y</kbd>       | Redo                                                                                                  |
+
+<br>
+<br>
+
+#### CONTEXT MENU AND DROP-DOWN MENU
+
+| Windows          | macOS            | Action                                          |
+| ---------------- | ---------------- | ----------------------------------------------- |
+| <kbd>Down↓</kbd> | <kbd>Cmd</kbd> + <kbd>Enter</kbd> | Move to the next option of the context menu     |
+
+<br>
+<br>
+
+#### MERGE CELLS
+
+| Windows          | macOS            | Action                                          |
+| ---------------- | ---------------- | ----------------------------------------------- |
+| <kbd>Ctrl</kbd> + <kbd>M</kbd> | <kbd>Cmd</kbd> + <kbd>M</kbd> | Merge selected cells     |
+
+## Managing keyboard shortcuts
+
+It is possible to add, change and remove keyboard shortcuts by managing the registered actions programmatically using the [Shortcut Manager](@/api/shortcut-manager) API.
+
+Each keyboard shortcut action is registered in a particular context. There are three contexts:
+
+- `grid` - activated when the user is browsing the data grid (initial)
+- `editor` - activated when the user opens a cell editor
+- `menu` - activated when the user opens the cell's context menu
+
+When the user interacts presses a key or a key combination keyboard, only the actions registered for the active context are executed. Only one context is active at a time.
+
+Please keep in mind that you can get already registered contexts
+using `getContext` method and even register new context using `addContext`. To switch to another context use `setActiveContextName` method.
+
+### Removing keyboard shortcuts
```

Comment:
@kirszenbaum I think it would be more natural to move "Removing keyboard shortcuts" after "Adding custom keyboard shortcuts"